app/models/exam.rb

```diff
@@ -28,4 +31,8 @@ def add_question question
   def remove_question question
     question.delete question
   end
+
+  def cal_time_limit
+    self.time_limit = hours.to_i * 3600 + minutes.to_i * 60
```

Comment:
```suggestion
    time_limit = hours.to_i * 3600 + minutes.to_i * 60
```
số cho vào Settings đi em